apcd-cms/src/apps/submissions/views.py

```diff
@@ -66,17 +65,22 @@ def filtered_submissions(self, submission_content, status, sort):
         def getDate(submission):
             date = submission['received_timestamp']
             return parser.parse(date) if date is not None else parser.parse('1-1-3005')
-
+        
+        ## Initial API response sort
         submission_list = sorted(
             submission_content,
             key=lambda row: getDate(row),
             reverse=(sort == 'Newest Received')
         )
-
         if status != 'All':
             submission_list = [submission for submission in submission_content 
```

Comment:
this can be renamed as submission_content and rest of the references and that should cover it.